.github/pull_request_template.md

```diff
@@ -19,6 +19,7 @@ Example how to mark a checkbox:-
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Code style update (formatting, local variables)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update.
```

Comment:
```suggestion
- [ ] Documentation update, if any is necessary
```

Reply:
I think we should keep it as-is because it's a checkbox that asks you for what you're doing in this PR. 😅